src/mito2/src/worker/handle_manifest.rs

```diff
@@ -158,6 +159,8 @@ impl<S> RegionWorkerLoop<S> {
         // Now the region is in editing state.
         // Updates manifest in background.
         common_runtime::spawn_global(async move {
+            WRITE_CACHE_INFLIGHT_DOWNLOAD.add(1);
```

Comment:
Should we add this gauge here?
https://github.com/GreptimeTeam/greptimedb/blob/5b1fca825a3eb1c07b6b528bec864ee1131e594d/src/mito2/src/cache/write_cache.rs#L198-L214